pgpool2-4.6.yaml

```diff
@@ -63,6 +68,59 @@ subpackages:
       runtime:
         - ${{package.name}}
 
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for Upstream pgpool image"
+    dependencies:
+      provides:
+        - pgpool2-compat=${{package.full-version}}
+      runtime:
+        - postgresql
+        - postgresql-client
+        - postgresql-dev
```

Comment:
Not convinced we need development headers for Postgres for pgpool. I'd omit these (even if upstream includes them) as they're useless outside of a build env

Reply:
Noted, updated PR accordingly.